semantics.json

```diff
@@ -342,6 +342,14 @@
         "default": true
       },
       {
+        "name": "randomizeOnRetry",
+        "type": "boolean",
```

Comment:
You're right. There's probably no case where someone would need to randomize the answers on retry only. And if someone asks, we could add that option later on as well. I'm going to change the behavior.